creator-node/src/index.ts

```diff
@@ -146,4 +179,11 @@ const startApp = async () => {
     }
   })
 }
-startApp()
+
+if (cluster.isMaster) {
```

Comment:
I think it's standard to not make it explicit (like don't set NODE_UNIQUE_ID to undefined) because it's undefined by default and then gets set for the workers. just made a note in start.sh and package.json -- good to know how it works